ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,424 @@
+
+    You can create a sha512_password by creating a username and password on a switch and copy/paste it within double quotes here.
+
+    ``` yaml
+    --8<--
+    examples/l2ls-fabric/group_vars/DC1.yml
+    --8<--
+    ```
+
+=== "DC1_FABRIC"
+    At the Fabric level (DC1_FABRIC), the following variables are defined in **group_vars/DC1_FABRIC.yml**. The fabric name, design type (l2ls), spine and leaf defaults, ansible authentication, and interface links are defined at this level. Other variables you must supply include: spanning-tree mode and priority along with an mlag IP pool.
+
+    Variables applied under the node key type (spine/leaf) defaults section are inherited to nodes under each type. These variables may be overwritten under the node itself.
+
+    The spine interface used by a particular leaf is defined from the leaf's perspective with a variable called `uplink_switch_interfaces`. For example, LEAF2 has a unique variable `uplink_switch_interfaces: [Ethernet2, Ethernet2]` defined. This means that LEAF2 is connected to SPINE1's Ethernet2 and SPINE2's Ethernet2, respectively.
+
+    ``` yaml
+    --8<--
+    examples/l2ls-fabric/group_vars/DC1_FABRIC.yml
+    --8<--
+    ```
+
+=== "DC1_SPINES"
+    In an L2LS design, there are two types of spine nodes: `spine` and `l3spine`. In AVD. the node type defines the functionality and the EOS CLI configuration to be generated. For an L2LS design, we will use node type: spine. Later, we will add routing to the Spines by changing the node type to l3spine.
+
+    ``` yaml
+    --8<--
+    examples/l2ls-fabric/group_vars/DC1_SPINES.yml
+    --8<--
+    ```
+
+=== "DC1_LEAFS"
+    In an L2LS design, we have 1 type of leaf node: `leaf`.
+
+    ``` yaml
+    --8<--
+    examples/l2ls-fabric/group_vars/DC1_LEAFS.yml
+    --8<--
+    ```
+
+=== "DC1_NETWORK_SERVICES"
+    You add VLANs to the Fabric by updating the **group_vars/DC1_NETWORK_SERVICES.yml**. Each VLAN will be given a name and a list of tags. The Tags can be used to filter the VLAN to specific Leaf Pairs (PODs). These variables are applied to both spine and leaf nodes since they are a part of this group.
+
+    ``` yaml
+    --8<--
+    examples/l2ls-fabric/group_vars/DC1_NETWORK_SERVICES.yml
+    --8<--
+    ```
+
+=== "DC1_NETWORK_PORTS"
+    Our fabric would not be complete without connecting some devices to it. We define connected endpoints and port profiles in **group_vars/DC1_NETWORKS_PORTS.yml**. Each endpoint's adapter defines which switch port(s) and port profile to use. In our example, we have 4 hosts and a firewall connected to the fabric. The connected endpoints keys are used for logical separation and apply to interface descriptions. These variables are applied to spine and leaf nodes since they are a part of this inventory group.
```

Comment:
```suggestion
    Our fabric would not be complete without connecting some devices to it. We define connected endpoints and port profiles in **group_vars/DC1_NETWORKS_PORTS.yml**. Each endpoint's adapter defines which switch port(s) and port profile to use. In our example, we have four hosts and a firewall connected to the fabric. The connected endpoints keys are used for logical separation and apply to interface descriptions. These variables are applied to spine and leaf nodes since they are a part of this inventory group.
```